source/install/install-rhel-71-postgresql.rst

```diff
@@ -0,0 +1,95 @@
+..  _install-rhel-71-postgresql:
+
+Installing PostgreSQL Database
+==============================
+
+1. Log into the server that will host the database, and open a terminal window.
+
+2. Download the PostgreSQL 9.4 Yum repository.
+
+  ``wget https://download.postgresql.org/pub/repos/yum/9.4/redhat/rhel-7-x86_64/pgdg-redhat94-9.4-3.noarch.rpm``
+
+3. Install the Yum repository from the file that you downloaded.
+
+  ``sudo yum localinstall pgdg-redhat94-9.4-3.noarch.rpm``
+
+4. Install PostgreSQL.
+
+  ``sudo yum install postgresql94-server postgresql94-contrib``
+
+5. Initialize the database.
+
+  ``sudo service postgresql initdb``
+
+6. Set PostgreSQL to start on boot.
+
+  ``sudo systemctl enable postgresql``
+
+7. Start the PostgreSQL server.
+
+  ``sudo systemctl start postgresql``
+
+8. Switch to the *postgres* Linux user account that was created during the installation.
+
+  ``sudo --login --user postgres``
+
+9. Start the PostgreSQL interactive terminal.
+
+  ``psql``
+
+10.  Create the Mattermost database.
+
+  ``postgres=# CREATE DATABASE mattermost;``
+
+11.  Create the Mattermost user 'mmuser'.
+
+  ``postgres=# CREATE USER mmuser WITH PASSWORD 'mmuser_password';``
+  
+  .. note::
+    Use a password that is more secure than 'mmuser-password'
+
+12.  Grant the user access to the Mattermost database.
+
+  ``postgres=# GRANT ALL PRIVILEGES ON DATABASE mattermost to mmuser;``
+
+13. Exit the PostgreSQL interactive terminal.
+
+  ``postgre=# \q``
+
+14. Log out of the *postgres* account.
```

Comment:
Ibid.

Say, did you know that ibid. is short for ibidem? The short form saves only one character.